sw/airborne/subsystems/imu/imu_mpu6000.c

```diff
@@ -28,12 +28,14 @@
 #include "subsystems/imu.h"
 #include "subsystems/abi.h"
 #include "mcu_periph/spi.h"
+#ifdef IMU_MPU_USE_MEDIAN_FILTER
```

Comment:
please replace the `#ifdef` by `#if` in this case, same for the other occurrences